tests/zopen_check_paxinstall

```diff
@@ -0,0 +1,154 @@
+#!/bin/sh
+
+#
+# FVT of zopen
+#
+#set -e # hard failure if any commands fail
+set -x
+WORKDIR="$1"
+
+fail(){ echo "$@"; exit 8; }
```

Comment:
I like this, perhaps we can make these functions available in a common script for consistency (could be in another PR)